ProjectTaskTracker.js

```diff
@@ -1,32 +1,37 @@
 
 const addNewProjectButton = document.getElementsByClassName("addButton")[0];
 const addTaskButton = document.getElementsByClassName("addButton")[1];
+
+const toDoList = document.getElementsByClassName("toDoList")[0];
+const inProgressList = document.getElementsByClassName('inProgressList')[0];
+const doneList = document.getElementsByClassName('doneList')[0];
 var toDoTaskNumber = 0;
 var projectNumber = 0;
 
 function addToDoTextarea() {
 	const newTask = document.createElement('textarea');
 	newTask.classList.add("createNewTask" + toDoTaskNumber);
+	newTask.classList.add("tasks");
+	newTask.setAttribute("draggable", "true");
 	return newTask;
 }
 
-function addToDoTask(){
-	const newTaskForm = document.createElement('textarea');
-	newTaskForm.classList.add("createNewTask" + toDoTaskNumber);
-	newTaskForm.classList.add("tasks");
-	newTaskForm.setAttribute("draggable", "true");
+function addNewProjectTextarea() {
+	const newProjectTextarea = document.createElement('textarea');
+	newProjectTextarea.classList.add("createNewProject" + projectNumber);
```

Comment:
I think you can add multiple in one call. E.g. `...classList.add("class1", "class2")`